engine/table/src/main/java/io/deephaven/engine/table/impl/replay/Replayer.java

```diff
@@ -186,6 +186,9 @@ public void setTime(long updatedTime) {
      */
     @Override
     public Table replay(Table dataSource, String timeColumn) {
+        if (dataSource.isRefreshing()) {
+            dataSource = TableTools.emptyTable(0).snapshot(dataSource);
+        }
```

Comment:
@rcaudy This is an interesting case.  Are you ok with this limitation, or is there a way to use a live table?

Reply:
Other comments sound like we are stuck with this limitation for the time being.

Reply:
> @rcaudy This is an interesting case.  Are you ok with this limitation, or is there a way to use a live table?

I think this code is entirely the wrong pattern if we wanted to have general updating tables as input. There are other tools for that. I think it's a fine limitation for replay to have.